src/pages/JavaThreadDumpAnalyzer.jsx

```
@@ -0,0 +1,332 @@
+      const uniqueThreadName = threadsTemp[threadNameWithState]
+        ? `${threadNameWithState} (${index})`
+        : threadNameWithState;
+      threadsTemp[uniqueThreadName] = { state, stackTrace: lines.join('\n') };
+    });
+    setThreadCounts(threadCountsTemp);
+    setThreads(threadsTemp);
+  };
+
+  const { getRootProps, getInputProps, isDragActive } = useDropzone({ onDrop });
+
+  const handleTextAreaChange = (e) => {
```

ESLint

[error] 'handleTextAreaChange' is assigned a value but never used.

(no-unused-vars)